web_poet/testing/utils.py

```diff
@@ -0,0 +1,44 @@
+import json
+import os
+from pathlib import Path
+from typing import Any, Iterable, Optional, Union
+
+from itemadapter import ItemAdapter
+
+from web_poet.serialization import SerializedDataFileStorage, serialize
+
+INPUT_DIR_NAME = "inputs"
+OUTPUT_FILE_NAME = "output.json"
+META_FILE_NAME = "meta.json"
```

Comment:
nitpick: it would probably make sense to move these constants closer to WebPoetItem